components/script/dom/htmlmediaelement.rs

```diff
@@ -269,6 +352,9 @@ impl HTMLMediaElement {
                         // Step 2.3.2.
                         this.upcast::<EventTarget>().fire_event(atom!("pause"));
 
+                        //FIXME(victor)
+                        //this.player.pause();
```

Comment:
This can be uncommented once we update servo-media to its latest revision (after #21325 lands)